src/commands/run.rs

```diff
@@ -253,6 +253,10 @@ pub struct RunCommand {
     /// memory, for example.
     #[clap(long)]
     trap_on_grow_failure: bool,
+
+    /// Run valgrind
```

Comment:
We'll want a much longer doc-comment of course, probably with a pointer to documentation for more details on how to use!